inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/util/InlongStreamSinkTransfer.java

```diff
@@ -207,6 +212,65 @@ private static StreamSink parseKafkaSink(KafkaSinkResponse sinkResponse, StreamS
         return kafkaSink;
     }
 
+    private static SinkRequest createHbaseRequest(StreamSink streamSink, InlongStreamInfo streamInfo) {
```

Comment:
Please add Java doc for the static method, thanks.

Reply:
done